helm_deploy/values-prod.yaml

```diff
@@ -13,11 +13,25 @@ generic-service:
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI: https://sign-in.hmpps.service.justice.gov.uk/auth/.well-known/jwks.json
     PRISON_API_ENDPOINT_URL: https://api.prison.service.justice.gov.uk
 
-# CloudPlatform AlertManager receiver to route promethues alerts to slack
-# See https://user-guide.cloud-platform.service.justice.gov.uk/documentation/monitoring-an-app/how-to-create-alarms.html#creating-your-own-custom-alerts
+  postgresDatabaseRestore:
+    enabled: true
+    namespace_secrets:
+      ma-rds-instance-output:
+        DB_NAME: "database_name"
+        DB_USER: "database_username"
+        DB_PASS: "database_password"
+        DB_HOST: "rds_instance_address"
+      ma-rds-instance-output-preprod:
```

Comment:
This was added a while ago with https://github.com/ministryofjustice/cloud-platform-environments/pull/6910